mllib/src/main/scala/org/apache/spark/mllib/clustering/PowerIterationClustering.scala

```diff
@@ -43,10 +43,12 @@ class PowerIterationClusteringModel(
  *
  * @param k Number of clusters.
  * @param maxIterations Maximum number of iterations of the PIC algorithm.
+ * @param initMethod Initialization method.
  */
 class PowerIterationClustering private[clustering] (
     private var k: Int,
-    private var maxIterations: Int) extends Serializable {
+    private var maxIterations: Int,
+    private var initMethod: String = "random") extends Serializable {
```

Comment:
We don't put the default values here but in the auxiliary constructor (for Java compatibility when we make it public) Remember to document the default value in the auxiliary constructor.